doc/sphinx/source/env_variables.rst

```diff
@@ -33,6 +33,11 @@ pocl.
  CL_INVALID_VALUE. If clock_gettime is available, messages
  will include a timestamp.
 
+ The old way (setting POCL_DEBUG to 1) has been updated to support categories.
```

Comment:
OK. Should be fine. The flag system caught my eye as it sort of "breaks the modularity" bringing things as "hsa" and "tce" to a central piece of code. But this is not a big deal as it's only a logger.